requirements.txt

```diff
@@ -22,7 +22,7 @@ croniter==0.3.30
 cryptography==2.7
 decorator==4.4.0          # via retry
 defusedxml==0.6.0         # via python3-openid
-flask-appbuilder==2.2.1
+flask-appbuilder==2.2.2rc1
```

Comment:
The latest rc is rc3. When no more features are needed to support this I'll release 2.2.2